### DISCUSSION TITLE
Is it CPython copy on Rust?

### DISCUSSION THREAD

**Discussion by Fidanze** [RustPython/RustPython#3943] [General]
i mean it will be absolute copy with WASM support?
maybe you will take some features from other implementations too? 
for example:
 tasklets like in stackless python,
 jit like in pypy

**Comment by youknowone** [comment]
We are heavily referring CPython, but never was limited to CPython.
We are looking for opportunities to support more stuff. There could be many things including your examples - though we don't have much yet.
